src/main/java/com/wavefront/WavefrontNozzleApplication.java

```diff
@@ -0,0 +1,17 @@
+package com.wavefront;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * Wavefront Nozzle Spring Boot Application
+ *
+ * @author Sushant Dewan (sushant@wavefront.com).
+ */
+@SpringBootApplication
+public class WavefrontNozzleApplication {
+
+  public static void main(String[] args) {
```

Comment:
we don't normally do indenting like this, please use the wavefront intellij style